android/src/main/java/com/tailscale/ipn/ui/viewModel/IpnViewModel.kt

```diff
@@ -208,6 +220,24 @@ open class IpnViewModel : ViewModel() {
 
   // Exit Node Manipulation
 
+  class AdvertisedRoutesHelper() {
```

Comment:
Can you please move `AdvertisedRoutesHelper` out to a dedicated file?

Reply:
Can you explain why? `AdvertisedRoutesHelper` is only used in `IpnViewModel`, so keeping it here makes it easier to understand the context it's used in

Reply:
It makes it easier to search for things across the repo. For the iOS project we follow a "one type for one file" rule, which means you can look up the file containing a given type without using full search.

Reply:
Done.